### ISSUE TITLE
Folder art not behaving properly. 228x204 squishes the image.

### ISSUE THREAD

**Issue by KedrenAderac** [ClusterM/hakchi2#1054]
Meaning if I use 228x204 image for my Box art, it will fill up the whole "cover art" area ... however using the very same image for Folders or similar dimensions will make the Folder icon squished. 

Is this intentional behavior? 
If so, is there a workaround to make Folder icons also fill up the whole "cover art" area?

**Comment by ReyVGM** [comment]
Try 228x228

**Comment by KedrenAderac** [comment]
@ReyVGM : Nah, doesn't work ... it's basically the same as the 204 since the aspect ratio doesn't change. It will fill up the "cover art" area vertically just fine like the 204 but there are gaps on the sides.

However people have asked about this before apparently and I managed to do what I wanted to do by overriding the image manually via FTP.
